repository_service_tuf/helpers/api_client.py

```diff
@@ -1,20 +1,25 @@
 # SPDX-FileCopyrightText: 2022-2023 VMware Inc
 #
 # SPDX-License-Identifier: MIT
-
+import time
 from dataclasses import dataclass
 from enum import Enum
 from typing import Any, Dict, Optional
 
 import requests
+from requests.exceptions import ConnectionError
+from rich.console import Console
 
 from repository_service_tuf.cli import click
 
+console = Console()
+
 
 class URL(Enum):
     token = "api/v1/token/"  # nosec bandit: not hard coded password.
     bootstrap = "api/v1/bootstrap/"
     task = "api/v1/task/?task_id="
+    publish_targets = "api/v1/targets/publish/"
```

Comment:
From what I know the enum types should be named as constants with all upper case.
This is also supported by the python docs on enums:
https://docs.python.org/3/library/enum.html
Do you think it's worth discussing in another issue?

Reply:
Issue created
- https://github.com/vmware/repository-service-tuf-cli/issues/197

Reply:
Issue filed with coment referencing both discussions.

- https://github.com/vmware/repository-service-tuf-cli/issues/199